src/Adapter/Product/Stock/CommandHandler/UpdateProductStockInformationHandler.php

```diff
@@ -91,5 +98,10 @@ public function handle(UpdateProductStockInformationCommand $command): void
             ),
             $command->getShopConstraint()
         );
+
+        dump($command);
```

Comment:
@FabienPapet 
I see a dump 😊

Reply:
ahahha :) fixed

Reply:
Eye of the tiger!

Reply:
ahaha :D 

the next thoughts is, can we include a check for this things in the automated test?